overrides/README.md

```diff
@@ -0,0 +1,11 @@
+# Customizing Paperless
```

Comment:
This is a great idea, and something I really need to remember to more of: putting documentation *in* the file structure to give people "signposts" rather than just assuming they've read all the documentation.  Thanks for this.